job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/EmbeddedApiDocGenerator.groovy

```diff
@@ -66,7 +66,7 @@ class EmbeddedApiDocGenerator {
         // remove HTML tags
         cleanText = cleanText.replaceAll(/<.*?>/, '')
         // unescape HTML and remove leading/trailing spaces
-        cleanText = unescapeHtml(cleanText).trim()
+        cleanText = unescapeHtml3(cleanText).trim()
```

Comment:
This needs a new dependency on:

```
<dependency>
    <groupId>io.jenkins.plugins</groupId>
    <artifactId>commons-text-api</artifactId>
</dependency>
```

It's probably passing the tests now by accident through a transitive dependency, but that could break in the future if that transitive dependency happens to change from underneath us. Bringing it in explicitly removes that risk.